components/Input/AmountInput.tsx

```diff
@@ -1,7 +1,7 @@
 import { Button } from "components";
 import UmaToken from "public/assets/icons/uma-token.svg";
 import styled from "styled-components";
-import { useOnChange } from "hooks";
+import { useHandleDecimalInput } from "hooks";
```

Comment:
Renamed this for clarity. David suggested this ages ago and it somehow slipped through the cracks.